app/CineHub/app/src/main/java/org/cinehub/BillBoardActivity.java

```diff
@@ -0,0 +1,27 @@
+package org.cinehub;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+
+import org.cinehub.utils.MovieAdapter;
+
+public class BillBoardActivity extends AppCompatActivity {
+
+    private MovieAdapter adapter;
+    private RecyclerView rv_movies;
```

Comment:
This variable should be written in camelCase as it's a Java var.

Reply:
Fix!